web/frontend/src/components/navigation/components/layer-three.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang='ts'>
```

Comment:
[nit] is this file concerned with laying obstacles on top of the map?
if so can the file name be changed to `obstacle-layer.svelte`?

Reply:
Yep! I like that name better. Renamed.